webpack.config.js

```diff
@@ -216,9 +204,6 @@ const webpackConfig = {
 			name: false,
 		},
 	},
-	node: {
```

Comment:
Afaict we don't need this any more. I'll confirm this by doing a bundle analyze to make sure we don't somehow bundle the crypto package.